pkg/cloudprovider/providers/rackspace/rackspace.go

```diff
@@ -391,3 +466,146 @@ func (os *Rackspace) GetZone() (cloudprovider.Zone, error) {
 
 	return cloudprovider.Zone{Region: os.region}, nil
 }
+
+// Create a volume of given size (in GiB)
+func (os *Rackspace) CreateVolume(name string, size int, tags *map[string]string) (volumeName string, err error) {
+	return "", errors.New("unimplemented")
+}
+
+func (os *Rackspace) DeleteVolume(volumeName string) error {
+	return errors.New("unimplemented")
+}
+
+// Attaches given cinder volume to the compute running kubelet
+func (os *Rackspace) AttachDisk(diskName string) (string, error) {
```

Comment:
elsewhere in this module `os` is used, I kept it for consistency, as I don't want to touch a lot of unrelated lines. I'll name it `rs` where I touch code, anyway. The naming issues should be addressed in a separate PR i guess.